### PR TITLE
metaphlan  v4.0.5 -> v4.0.6

### PR DESCRIPTION
**Pull request by tuncK**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Updated metaphlan version, commented out DM test as the target file is too big to be used as a test.

### CODE REVIEW

File: tools/metaphlan/macros.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <macros>
-    <token name="@TOOL_VERSION@">4.0.5</token>
+    <token name="@TOOL_VERSION@">4.0.6</token>
```
Comment (bernt-matthias) [inline tools/metaphlan/macros.xml:3]: Apparently tests for the DM do not run. Not sure why. 

Will update the tool in https://github.com/galaxyproject/tools-iuc/pull/5175

```suggestion
    <token name="@TOOL_VERSION@">4.0.5</token>
```

**Review by bernt-matthias — APPROVED** [review]
Hi @tuncK and @bgruening 

Fine from my side. CI considers the tool now .. but at the same time all tests have been commented. 

Guess there is nothing else that we can do, or?

What is the limiting factor in testing: time or disk space?

**Comment by bgruening** [comment]
As far as I understood its a space limitation. I also have no better idea. Please feel free to merge.

File: data_managers/data_manager_metaphlan_database_downloader/data_manager/data_manager_metaphlan_download.xml [hunk 2]
```diff
@@ -35,6 +36,7 @@ python '$__tool_directory__/data_manager_metaphlan_download.py'
                 </assert_contents>
             </output>
         </test>
+        -->
```
Comment (bernt-matthias) [inline data_managers/data_manager_metaphlan_database_downloader/data_manager/data_manager_metaphlan_download.xml:39]:
```suggestion
```

File: data_managers/data_manager_metaphlan_database_downloader/data_manager/data_manager_metaphlan_download.xml [hunk 3]
```diff
@@ -26,6 +26,7 @@ python '$__tool_directory__/data_manager_metaphlan_download.py'
         <data name="out_file" format="data_manager_json" label="${tool.name}"/>
     </outputs>
     <tests>
+        <!--
```
Comment (bernt-matthias) [inline data_managers/data_manager_metaphlan_database_downloader/data_manager/data_manager_metaphlan_download.xml:29]:
```suggestion
```

**Comment by bernt-matthias** [comment]
> As far as I understood its a space limitation. I also have no better idea. Please feel free to merge.

yep to large. 2GB .. needing ~200min

**Pull request merged**